### PR TITLE
Added Food master Project

### PR DESCRIPTION
**Pull request by Surya8991**
### 🛠️ Fixes Issue (Number)
Issue Number #808 | SSOC 2022 Participant

### 👨‍💻 Changes proposed

I have added a new Project named as Food Master

### ✅ Check List (Check all the applicable boxes) 

- [x] My code doesn't break any part of the project (Zero Octave-Javascript-Projects).
- [x] This PR does not contain plagiarized content.
- [x] My Addition/Changes works properly and matches the overall repo pattern.
- [x] The title of my pull request is a short description of the requested changes.

### 📄 Note to reviewers

Do check the Project and tell me if any changes are needed.
### 📷 Screenshots

image
image

### CODE REVIEW

**Review by myhacktech — APPROVED** [review]

**Review by myhacktech — APPROVED** [review]
nice work

**Comment by Surya8991** [comment]
Will it merge now?

**Comment by myhacktech** [comment]
> Will it merge now?

will be done by PA

**Comment by Surya8991** [comment]
Ok tks

File: cards.json [hunk 1]
```diff
@@ -90,6 +90,11 @@
         "image": "assets/Images/calculator.jpg",
         "link": "Public/calculator.html"
     },
+    {
```
Comment (Astrodevil) [inline cards.json:93]: Add in alphabetical order & also add your project in README. ANswer me, If you can create a README about this project.

Comment (Surya8991) [reply cards.json:93]: Yeah I will add in alphabetic order and about readme I will try to add and tell u if I cannot add.

Comment (Astrodevil) [reply cards.json:93]: okh I will, but give it a try once

Comment (Surya8991) [reply cards.json:93]: Sure do give me 10min I will make a readme having dinner now.

Comment (Astrodevil) [reply cards.json:93]: No hurry, do it anytime or day. SSOC is 3 month long. :)

Comment (Surya8991) [reply cards.json:93]: But I have a doubt like how can add ReadMe for this now?

**Pull request merged**